sources/packages/web/src/store/modules/student/getters.ts

```diff
@@ -0,0 +1,7 @@
+import { GetterTree } from "vuex";
+import { RootState } from "@/types";
+import { StudentState } from "./student";
```

Comment:
import using @/

Reply:
This one I believe does not make sense. There is a file structure that is replicated and if moved should be moved as a whole, as shown below. Does it make sense?
![image](https://user-images.githubusercontent.com/61259237/170128488-ae8c8db0-67da-4969-8daf-99c44e4ff2c9.png)

Reply:
Ah.. Yes.